ProcessMaker/processmaker
# Feature/FOUR-20299: List of Processes in Home Page - Implement Process list pane collapse

## Issue & Reproduction Steps
In this ticket we have the next new features:
- new Participant Home Screen Inbox component:
  * index.blade.index(Tasks) keeps old functionality  and also calls new component ParticipantHomeScreen ( old and new features were divided with div's and are shown or hidden with variable showOldTaskScreen=true/false
  * New Collapsable List Pane (Currently empty, the functionality for Process and Dashboards will be added in future tickets)
  * Collapsable side-bar Status(open or collapsed) is saved in database per User
  * New Component has all Task Page functionality

## How to Test
- Login PM4
- Go to TASKS
- New Participant Home Inbox should be displayed with left Pane Opened
- Click in Collapse Icon to hide Left Pane
- Switch to another page like Admin  and return to Tasks (last status for collapsable Pane should be applied)

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-20299

## Code Review Checklist
